app_data/vocabularies/resource_types.yaml

```diff
@@ -0,0 +1,1057 @@
+    cs: Patent
+  tags:
+    - depositable
+    - linkable
+- id: publication-peerreview
+  icon: file alternate
+  props:
+    csl: article
+    datacite_general: PeerReview
+    datacite_type: ""
+    openaire_resourceType: "0015"
+    openaire_type: publication
+    eurepo: info:eu-repo/semantics/review
+    schema.org: https://schema.org/ScholarlyArticle
+    subtype: publication-peerreview
+    type: publication
+    marc21_type: publication
+    marc21_subtype: peerreview
+  title:
+    en: Peer review
+    sv: Peer review
+    de: Peer Review
+    cs: Recenze
+  tags:
+    - depositable
+    - linkable
+- id: publication-preprint
+  icon: file alternate
+  props:
+    csl: article
+    datacite_general: Preprint
+    datacite_type: ""
+    openaire_resourceType: "0016"
+    openaire_type: publication
+    eurepo: info:eu-repo/semantics/preprint
+    schema.org: https://schema.org/ScholarlyArticle
+    subtype: publication-preprint
+    type: publication
+    marc21_type: publication
+    marc21_subtype: preprint
+  title:
+    en: Preprint
+    sv: Preprint
+    de: Preprint
+    cs: Preprint
+  tags:
+    - depositable
+    - linkable
+- id: publication-deliverable
+  icon: file alternate
+  props:
+    csl: report
+    datacite_general: Text
+    datacite_type: Project deliverable
+    openaire_resourceType: "0034"
+    openaire_type: publication
+    eurepo: info:eu-repo/semantics/report
+    schema.org: https://schema.org/CreativeWork
+    subtype: publication-deliverable
+    type: publication
+    marc21_type: publication
+    marc21_subtype: deliverable
+  title:
+    en: Project deliverable
+    sv: Projektresultat
+    de: Projektergebnis
+    cs: Výstup projektu
+  tags:
+    - depositable
+    - linkable
+- id: publication-milestone
+  icon: file alternate
+  props:
+    csl: report
+    datacite_general: Text
+    datacite_type: Project milestone
+    openaire_resourceType: "0035"
+    openaire_type: publication
+    eurepo: info:eu-repo/semantics/report
+    schema.org: https://schema.org/CreativeWork
+    subtype: publication-milestone
+    type: publication
+    marc21_type: publication
+    marc21_subtype: milestone
+  title:
+    en: Project milestone
+    sv: Milstolpe för projektet
+    de: Projektmeilenstein
+    cs: Mezník projektu
+  tags:
+    - depositable
+    - linkable
+- id: publication-proposal
+  icon: file alternate
+  props:
+    csl: article
+    datacite_general: Text
+    datacite_type: Proposal
+    openaire_resourceType: "0036"
+    openaire_type: publication
+    eurepo: info:eu-repo/semantics/researchProposal
+    schema.org: https://schema.org/CreativeWork
+    subtype: publication-proposal
+    type: publication
+    marc21_type: publication
+    marc21_subtype: proposal
+  title:
+    en: Proposal
+    sv: Förslag
+    de: Antrag
+    cs: Návrh
+  tags:
+    - depositable
+    - linkable
+- id: publication-report
+  icon: file alternate
+  props:
+    csl: article
+    datacite_general: Report
```

Comment:
Duplication? Given that we now have Text / Report, I will only have one

Reply:
see my comment regarding the parent type, I think we could leave report here